lib/sgtn-client/api/source.rb

```diff
@@ -38,8 +44,11 @@ def self.getSources(component, locale)
         return items
       end
 
-      def self.loadBundles(locale)
+      def self.loadBundles(locale="en")
         SgtnClient.logger.debug "[Source][loadBundles]locale=#{locale}"
+        if locale.nil?
+          locale="en"
```

Comment:
Do not hardcode "en" here (or anywhere). Set and get the default from SgtnClient::Config.configurations.

Reply:
calling this method(loadBundles) is prior to loading the configuration, so has to hardcode it in the codes.